package.json

```diff
@@ -37,7 +37,11 @@
     "release:all": "npm run setenv && npm run build:mac && npm run build:win && npm run build:linux",
     "build:mac": "npm run pack:mac && build \"dist/mac/ngrev-darwin-x64/ngrev.app\" --platform=mac",
     "build:win": "npm run pack:win && build \"dist/win/ngrev-win32-ia32\" --platform=win",
-    "build:linux": "npm run pack:linux && build \"dist/win/ngrev-linux-x64\" --platform=linux"
+    "build:linux": "npm run pack:linux && build \"dist/win/ngrev-linux-x64\" --platform=linux",
+    "vscode:prepublish": "npm run compile",
```

Comment:
Probably this should be `vscode:compile`?

Reply:
Well, maybe yes. I just copied it straight through the extensions tutorial 😄